portfolio/src/main/webapp/script.js

```diff
@@ -27,29 +27,23 @@ function addRandomFunFact() {
   funfactContainer.innerText = funfact;
 }
 
-function getRandomQuote() {
-  fetch('/data').then(response => response.json()).then((quotes) => {
+//Saving code in case needed for step 5.
 
-  const quoteListElement = document.getElementById('quote-container');
-  quoteListElement.innerHTML = '';
-  quoteListElement.appendChild(
-      createQuoteElement('English: ' + quotes.English));
-  console.log(quotes.English);
-  quoteListElement.appendChild(
-      createQuoteElement('Spanish: ' + quotes.Spanish));
-  console.log(quotes.Spanish);
-  quoteListElement.appendChild(
-      createQuoteElement('French: ' + quotes.French));
-  console.log(quotes.French);
-  quoteListElement.appendChild(
-      createQuoteElement('Hawaiian: ' + quotes.Hawaiian));
-  console.log(quotes.Hawaiian);
-  });
-}
+// function getComment() {
+//   fetch('/data').then(response => response.json()).then((entries) => {
 
- function createQuoteElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
-}
+//   const quoteListElement = document.getElementById('entry-container');
+//   quoteListElement.appendChild(
+//       createQuoteElement('Name: ' + entries.name));
+//   quoteListElement.appendChild(
+//       createQuoteElement('Email: ' + entries.email));
+//   quoteListElement.appendChild(
+//       createQuoteElement('Comment: ' + entries.comment));
+//   });
+// }
 
+//  function createQuoteElement(text) {
+//   const liElement = document.createElement('li');
+//   liElement.innerText = text;
+//   return liElement;
+//  }
```

Comment:
You can uncomment and leave the function as is, if it going to be called in a follow up PR. You can add a TODO to the function saying "Use function in step 5 to display comment"

Reply:
Done